app/src/test/java/nl/tudelft/b_b_w/model/BlockUnitTest.java

```diff
@@ -158,6 +158,32 @@ public void testTrustValueInit() {
     }
 
     /**
+     * Test to check whether the verifyBlock() method returns the right boolean value indicating if this block is equal to the parameter block.
+     * @throws Exception Catches error when the MessageDigest
+     * gets an error.
+     */
+    @Test
+    public void verifyBlockTest() throws Exception {
+        final Block check = BlockFactory.getBlock(TYPE_BLOCK, owner, blockController.getLatestSeqNumber(owner)+1,
```

Comment:
The function in the BlockController is not correct, so until it has been refactored, it is discouraged to use it.